MatrixSDK/JSONModels/MXEvent.m

```diff
@@ -196,20 +196,30 @@ - (NSUInteger)age
     return age;
 }
 
-- (NSDictionary *)originalDictionary
+- (NSDictionary *)JSONDictionary
```

Comment:
Note returned dictionary is slightly different from the original JSON here because it does not consider 'unsigned' key introduced in v2.